contributing.md

```diff
@@ -63,6 +63,10 @@ class Foo:
         return session.run(incr_tensor)
 ```
 
+### Naming conventions
+
+Variable names: scalars and vectors start lowercase, but following the notation used in Gaussian process papers, all matrices are denoted with upper case. For example, `lengthscales` denotes a vector, whereas `Xnew` denotes a matrix.
```

Comment:
I still get confused with (N,1)  is a matrix and (N,) is a vector. Perhaps worth calling out in the guide what you mean by the difference between vector and matrix.

Reply:
clarified. better?